### ISSUE TITLE
[profiles] Use foreign category for newznab indexers when searching for foreing languages

### ISSUE THREAD

**Issue by smitetierlist** [Sonarr/Sonarr#855]
TL:DR; **Foreign category is missing for usenet indexers when searching for foreign languages**

Okay I'm not sure how I would implement this, anyway:

I had a very hard time figuring out why sonarr wasn't finding almost none episodes. After checking my indexer I realized there are enough matches that should fit.

The problem was the foreign category (`5020` usually) was missing in my indexer settings.

I think there should be some *automation*. When I'm looking for foreign content the category should be set up automatically. 

The obvious problem is that the indexer setting is independent from the profile. Maybe we could use some simple checkbox like

    [  ] Search also for foreign content

And in advanced settings there could be an additional `input` to edit the foreign category.

**Comment by thezoggy** [comment]
the problem is that there is just so much 'crap' in foreign. also they use different naming schemes.. so the standard regex may not work even if your looking in the cat. ideally you need to change out the naming exceptions/show names/quality format(regex)/and etc. then also when searching/handling the releases you need to not mix foreign with non-foreign. like what if you grabbed x german release which uses a different season/ep scheme than the french one? and what if one was a repack/proper.. etc can get pretty complicated quickly.

**Comment by smitetierlist** [comment]
> the problem is that there is just so much 'crap' in foreign

Well. In non-foreign categories is no foreign content. At least almost none.

**Comment by markus101** [comment]
The problem is the same as anime, there is no guarantee the same category ID is used, checking two different indexers gave me two different results. In addition Sonarr has no confirmation that the series is foreign and we'd need to support another series type to be absolutely sure. I don't see this complexity being worth the effort at this time.

**Comment by tostane** [comment]
just add a category per show  have them default to the normal but allow us to put in 5020 on those that need it.
